[agent builder] Document the workflows tool type (#4717)
elastic/docs-content: explore-analyze/ai-features/agent-builder/workflows.md

---
navigation_title: "Use workflows"
description: "Learn how to trigger workflows from agents and invoke agents within workflow steps."
applies_to:
  stack: preview =9.2, ga 9.3+
  serverless:
    elasticsearch: ga
    observability: preview
    security: preview
products:
  - id: elasticsearch
  - id: kibana
  - id: observability
  - id: security
  - id: cloud-serverless
---

# Integrate workflows with {{agent-builder}}

> {{agent-builder}} integrates bi-directionally with [Elastic Workflows](/explore-analyze/workflows.md). This integration bridges the gap between conversational reasoning and automated execution:

Reviewer: ```suggestion
Agent Builder is tightly integrated with [Elastic Workflows](/explore-analyze/workflows.md):
```

I think we can simplify here :)
